src/components/v5/common/MemberSignature/types.ts

```diff
@@ -0,0 +1,6 @@
+import { AvatarUserProps } from '~v5/shared/AvatarUser/types';
+
+export interface MemberSignatureProps
+  extends Omit<AvatarUserProps, 'size' | 'isLink'> {
```

Comment:
Please add a prop called avatarProps and use this interface there. It will be easier to update this component later if needed